.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+For any feature request, please use the [forum](https://forum.cozy.io/c/francais/banque), thanks !
```

Comment:
Don't we want to use issues templates instead of PR templates ? https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Reply:
Thanks, fixed

Reply:
Ici https://github.com/cozy/cozy-banks/pull/1772/files